packages/flutter_tools/test/commands.shard/permeable/build_bundle_test.dart

```diff
@@ -139,6 +139,26 @@ void main() {
     FeatureFlags: () => TestFeatureFlags(),
   });
 
+  testUsingContext('bundle fails to build for Windows arm64 if feature is disabled', () async {
```

Comment:
how do you enable this feature?

Reply:
oh, just the windows desktop feature?

Reply:
Yes.
I updated those tests title to
`bundle fails to build for Windows if feature is disabled (on {x64, arm64} platform)` instead of previous version, which should be more clear about the fact we mention Windows feature (and not the platform support).